woocommerce/woocommerce
# Add a margin to the bestseller widget product title

### All Submissions:






### Changes proposed in this Pull Request:

This PR fixes an overlap issue when the product title of the best selling product is too long and then overlaps the sparkline in the bestseller widget.

Ran grunt css which recompiled more files than what has changed hench all the inclusions in the PR.

Closes #20838.

### How to test the changes in this Pull Request:

1. Edit the bestselling product and make the title very long.
2. View the dashboard and it should not overlap as per #20838
3. Example http://cld.wthms.co/CEIwmR

### Other information:




### Changelog entry

> Tweak - Add margin to best selling product title in the dashboard widget to avoid overlapping the sparkline.
